### PR TITLE
minor fix and cleaning changes

### PR DESCRIPTION
**Pull request by etienne-lms**

### CODE REVIEW

File: module/psu/src/mod_psu.c [hunk 1]
```diff
@@ -358,6 +358,9 @@ static int mod_psu_process_bind_request(
         *api = &mod_psu_driver_response_api;
 
         break;
+
+    default:
+	fwk_unexpected();
```
Comment (vijayenthiran-arm) [inline module/psu/src/mod_psu.c:363]: good to add `break;` here?

Comment (etienne-lms) [reply module/psu/src/mod_psu.c:363]: true.

**Review by jimqui01 — APPROVED** [review]
jenkins: ok to test

**Review by tarek-arm — DISMISSED** [review]
jenkins: ok to test

**Comment by etienne-lms** [comment]
Fixed missing `break;` in mod_psu.c + fix 2 indentation (replace tabs with space chars).

File: module/psu/src/mod_psu.c (hunk 1 above)
Comment (etienne-lms) [reply module/psu/src/mod_psu.c]: Done

**Comment by jimqui01** [comment]
jenkins: ok to test

**Comment by jimqui01** [comment]
processing mod_i2c.c
325:/work/workspace/workspace/scp-v2-tests-ci/scp/module/i2c/src/mod_i2c.c has tab

**Comment by etienne-lms** [comment]
deprecated

**Pull request closed without merging**